modules/gltf/editor_scene_exporter_gltf_plugin.cpp

```diff
@@ -64,8 +64,8 @@ SceneExporterGLTFPlugin::SceneExporterGLTFPlugin() {
 	file_export_lib->add_filter("*.glb");
 	file_export_lib->add_filter("*.gltf");
 	file_export_lib->set_title(TTR("Export Mesh GLTF2"));
-	String gltf_scene_name = TTR("Export GLTF...");
-	add_tool_menu_item(gltf_scene_name, callable_mp(this, &SceneExporterGLTFPlugin::convert_scene_to_gltf2));
+	String gltf_scene_name = TTR("GLTF...");
```

Comment:
I'd suggest being more explicit, and using the correct casing for glTF:

```suggestion
	String gltf_scene_name = TTR("glTF 2.0 Scene...");
```

Reply:
Done.